package.json

```diff
@@ -4,31 +4,36 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "prebuild": "npm run fix && npm run coverage",
+    "build": "next build --no-lint",
     "start": "next start",
-    "lint": "gts lint",
```

Comment:
use `eslint` directly instead of `gts` to ensure that we also applies Next's `eslint` rules